lib/segment/src/vector_storage/async_raw_scorer.rs

```diff
@@ -69,45 +70,17 @@ impl<TQueryScorer> RawScorer for AsyncRawScorerImpl<'_, TQueryScorer>
 where
     TQueryScorer: QueryScorer<[VectorElementType]>,
 {
-    fn score_points(&self, points: &[PointOffsetType], scores: &mut [ScoredPointOffset]) -> usize {
+    fn score_points(&self, scores: &mut [ScoredPointOffset]) {
         if self.is_stopped.load(Ordering::Relaxed) {
-            return 0;
+            return;
```

Comment:
Not sure if this is correct: previous implementation explicitly returned no scored points. Current implementation, looks like, returns points with unassigned scores